src/app/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -0,0 +1,7 @@
+"use client";
+
+import { SignIn } from "@clerk/nextjs";
+
+export default function SignInPage() {
+  return <SignIn />;
+}
```

Comment:
The applications in Clerk for the AI project have been configured to redirect users to `/sign-in` and `/sign-up` in the host app. So I had to go ahead and create these pages to get things working — we will need these to properly brand them but I needed to bring this initial work forward.